{"name": "Невозможность изменения данных неавторизованного пользователя, password", "status": "passed", "description": "При попытке изменения {field} возвращает HTTP 401 и сообщение об ошибке", "steps": [{"name": "Отправка POST на auth/login", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'bakerlaura@example.com', 'password': '3065330047', 'name': 'Christina Owen'}"}], "start": 1718207481641, "stop": 1718207481906}, {"name": "Отправка PATCH на auth/user", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'bakerlaura@example.com', 'password': 'abracadabra', 'name': 'Christina Owen'}"}, {"name": "access_token", "value": "''"}], "start": 1718207481906, "stop": 1718207482113}], "parameters": [{"name": "field", "value": "'password'"}], "start": 1718207481641, "stop": 1718207482114, "uuid": "ebffefaf-78f2-4d05-8b66-11294d538796", "historyId": "755912a3ed0a006aa8978bdca293b242", "testCaseId": "f450658db434b016c441fc12e73231b2", "fullName": "tests.test_auth_user.TestChangeUserData#test_change_user_data_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "XPMEPODW0067"}, {"name": "thread", "value": "9424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}